cogs/quotes.py

```diff
@@ -100,16 +100,26 @@ def rebuild_mc(self):
         conn.close()
 
     @commands.command(aliases=['addq'])
-    async def add_quotes(self, ctx, member: discord.Member, *, quote: str):
+    async def add_quotes(self,
+                         ctx,
+                         member: discord.Member = None,
+                         *,
+                         quote: str = None):
         """
         Add a quote to a user's quote database.
         """
-
+        replying: bool = ctx.message.reference and ctx.message.reference.resolved
+        if quote is None:
+            if not replying:
+                return
+            member = member or ctx.message.reference.resolved.author
+            quote = ctx.message.reference.resolved.content
         conn = sqlite3.connect(self.bot.config.db_path)
         c = conn.cursor()
         t = (member.id, member.name, quote, str(ctx.message.created_at))
         c.execute('INSERT INTO Quotes VALUES (?,?,?,?)', t)
-        msg = await ctx.send('`Quote added.`')
+        msg = await ctx.send(
+            f"`{quote}` added to `{member.display_name}`'s quotes.")
```

Comment:
i know we had a discussion about this and i cant remember what we decided, but doesn't this get a bit spammy?

Reply:
I'd say for this go with Quote added. surrounded by triple ticks cause i hate single tick appearance